src/SearchResults.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import TableRow from "./TableRow.js";
+import CustomerProfile from "./CustomerProfile.js";
+
+function SearchResults(props) {
+  const [profileId, setProfileId] = useState(0);
+  const [rows, setRows] = useState(props.results);
```

Comment:
On line 7 you are initalising some state using _props_ from the parent component. This is an anti-pattern in React, indeed it means that the component won't update the state in the way you expect. If in doubt  remember that the props are read-only and should change when they are passed through to the component from above.